routing/pathfind_test.go

```diff
@@ -72,13 +72,13 @@ var (
 
 	tlvPayAddrFeatures = lnwire.NewFeatureVector(
 		lnwire.NewRawFeatureVector(
-			lnwire.TLVOnionPayloadOptional,
+			lnwire.TLVOnionPayloadRequired,
 			lnwire.PaymentAddrOptional,
 		), lnwire.Features,
 	)
 
 	mppFeatures = lnwire.NewRawFeatureVector(
-		lnwire.TLVOnionPayloadOptional,
+		lnwire.TLVOnionPayloadRequired,
```

Comment:
I wonder if we should replace all occurrences of `HasFeature(lnwire.TLVOnionPayloadOptional` or `supports(lnwire.TLVOnionPayloadOptional)`) over to `HasFeature(lnwire.TLVOnionPayloadOptional`, just to ease future code review? Otherwise we might stumble over that code and think "but hey, I thought we made those mandatory" without realizing that `HasFeature` checks for either feature pair.

Just a nit though, non blocking!

Reply:
Maybe we can newtype the feature bit type (without optional/required) and require the newtype as the argument to `HasFeature`